airflow/models.py

```diff
@@ -4101,6 +4091,7 @@ def refresh_from_db(self, session=None):
 
         self.id = dr.id
         self.state = dr.state
+        self.is_paused = dr.is_paused
```

Comment:
Yeah you're right. THe perf is bad, but I confused dag with dagrun. So I'll probably just cache it.